content/millennium/bulk-data.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Bulk Data Access
+layout: api
+---
+
+# Bulk Data Access Overview
+
+The Cerner Ignite APIs for Millennium with Bulk Data Access provides system applications the ability to access large volumes of core clinical data for a group of individuals in alignment with standards cited in 21st Century Cures Act regulations. 
+
+FHIR Bulk data requests are executed against an organization's database asynchronously. When implementing an interface between a Cerner Millennium domain and a bulk data application, it is important that the queries inherent in the bulk data request capture only the data necessary for the application.
```

Comment:
I noticed that we have not mentioned the Service Root URL anywhere?

Reply:
I have added this section to the main Overview page: https://github.com/cerner/fhir.cerner.com/pull/899/commits/c936d35a3ba46f521cf0e5f6addfec2b7276c599#diff-b781e82e47439c1cc68cf3a6a51ab1668b5f40d37870022bec1936c5c69d6bc1R52-R61, which points to the service Root url section from the R4 overview.

Also updated the URL here and added FHIR Base: https://github.com/cerner/fhir.cerner.com/pull/899/commits/c936d35a3ba46f521cf0e5f6addfec2b7276c599#diff-c2a4803bbbf3b17e514d41cc6eb7d578c94a283f937cce0400f5437570744d7fR35